src/commands/run-tests.yml

```diff
@@ -23,6 +23,19 @@ parameters:
       semicolon-separated list.
     type: string
     default: ''
+  select-by-folder:
+    description: >
+      Location of the folder containing test code, relative to the project root folder. The
```

Comment:
How about this?

Location of the folder containing tests, relative to the project root folder. To generate a test suite, MATLAB uses only the tests in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list.

Reply:
Changed it.